site/content/docs/5.3/components/card.md

```diff
@@ -377,6 +407,13 @@ Similar to headers and footers, cards can include top and bottom "image caps"—
 
 Turn an image into a card background and overlay your card's text. Depending on the image, you may or may not need additional styles or utilities.
 
+{{< design-callout-alert >}}
+This variant should not be used because it does not respect the Orange Design System specifications. It does not respect accessibility requirements neither.
```

Comment:
```suggestion
This variant should not be used because it does not respect the Orange Design System specifications. It might lead to accessibility issues as well. Prefer using a card with a top image inside.
```